src/cli.ts

```diff
@@ -175,16 +175,18 @@ async function prepareSuites(inputs: string[]) {
    * Validate and handle configs
    */
   const config = readConfig(environment, options.config);
-  /**
-   * Favor playwright options passed via cli to inline playwright options
-   */
-  config.playwrightOptions = merge(config.playwrightOptions, options.playwrightOptions || {});
+
   const params = merge(
     config.params,
     options.suiteParams || {},
     options.params || {}
   );
+
+  /**
+   * Favor playwright options passed via cli to inline playwright options
+   */
   const playwrightOptions = merge(config.playwrightOptions, {
+    ...options.playwrightOptions,
```

Comment:
Apologies for nitpicking this. It would be better to do merge(config.playwrightOptions, options.playwrightOptions, {...}) than `...options,playwrightOptions` spreading would be slower. Not a huge deal, but good to keep in mind.